docs/markdown/Builtin-options.md

```diff
@@ -281,3 +282,12 @@ relative to the installation `prefix`, which will often result in installed pyth
 modules to not be found by the interpreter unless `prefix` is `/usr` on Linux,
 or for example `C:\Python39` on Windows. These options can be absolute paths
 outside of `prefix`.
+
+*Since 0.61.0* The `python.install_env` option is used to detect the correct
+installation path. Setting to `system` will avoid making the paths relative to
+`prefix`. Setting to `venv` will instead use the paths for the virtualenv the
```

Comment:
Also needs explanation of what `system` actually does do (see my comment on line 6 of `python_module_env.md`).